pkg/apiclient/workflowarchive/workflow-archive.proto

```diff
@@ -24,6 +24,13 @@ message ListArchivedWorkflowLabelKeysRequest {
 message ListArchivedWorkflowLabelValuesRequest {
     k8s.io.apimachinery.pkg.apis.meta.v1.ListOptions listOptions = 1;
 }
+message RetryArchivedWorkflowRequest {
```

Comment:
this should be same parameters as normal retry - can you please check

Reply:
This is the same parameters as a normal retry however it also includes the `uid` to be passed for `GetArchivedWorkflow`.